pkg/pillar/zboot/zboot.go

```diff
@@ -339,6 +339,12 @@ func WriteToPartition(log *base.LogObject, image string, partName string) error
 		log.Errorf("WriteToPartition failed %s\n", errStr)
 		return errors.New(errStr)
 	}
+	err := syscall.Unmount(devName, syscall.MNT_FORCE)
+	for err != nil {
+		errStr := fmt.Sprintf("Unmount of %s failed: %s", devname, err)
```

Comment:
That is normal. The case we want to log is when the umount succeeded because that would be odd